keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -881,8 +883,6 @@ in back-to-back releases.
 - Multiple examples of real world uses
 - Production support in both CRI-O and containerd
 - Allowing time for feedback
-- Consider a new `RuntimeConfig` field to indicate to end users if the feature
```

Comment:
I don't think we need this, because we would still have runtime versions supporting the feature but not the [`RuntimeFeatures`](https://github.com/kubernetes/cri-api/blob/479df365173e528979ac14855f456db9a0601cd5/pkg/apis/runtime/v1/api.proto#L1728-L1734) field. This would add another level of skew to the whole feature and seems unnecessary complicated.

Reply:
agreed